build/BuildSteps.cs

```diff
@@ -193,7 +193,9 @@ public static void FilterPowershellRuntimes()
         public static void FilterPythonRuntimes()
         {
             var minifiedRuntimes = Settings.TargetRuntimes.Where(r => r.StartsWith(Settings.MinifiedVersionPrefix));
-            foreach (var runtime in Settings.TargetRuntimes.Except(minifiedRuntimes))
+            var unsupportedPythonRuntimes = Settings.UnsupportedPythonRuntimes;
```

Comment:
I think this is a safe change, win-arm64 is not supported in python so not having a directory for this is OK.